### DISCUSSION TITLE
How to escape template syntax

### DISCUSSION THREAD

**Discussion by Archie-2021** [twpayne/chezmoi#3253] [Q&A] [answered]
So I started using templates from changing my lfrc file into a template. Turns out when adding a file as a template in chemoi it doesnt escape the similar syntax already written in file. I guess it is because lf also is written in go. 
How can i escape {{ }} syntax in my files?

**Comment by twpayne** [comment]
Use:

    {{ "{{" }}

and

    {{ "}}" }}

**Reply by bradenhilton** [reply, marked as answer]
Or [change the delimiters](https://www.chezmoi.io/reference/templates/directives/#delimiters) if applicable.

**Comment by Archie-2021** [comment]
> Or [change the delimiters](https://www.chezmoi.io/reference/templates/directives/#delimiters) if applicable.

Thanks , that is solved it very well.
